GherkinSyncTool/GherkinSyncTool/Synchronizers/TestRailSynchronizer/Client/TestRailClientWrapper.cs

```diff
@@ -102,5 +91,26 @@ public IEnumerable<Case> GetCases(ulong projectId, ulong suiteId)
             ValidateRequestResult(result);
             return result.Payload;
         }
+
+        private static bool CompareTestCaseContent(CreateCaseRequest createCaseRequest, Case testRailCase)
+        {
+            if(!testRailCase.Title.Equals(createCaseRequest.Title)) return true;
+            if(!testRailCase.TemplateId.Equals(createCaseRequest.TemplateId)) return true;
```

Comment:
One return statement should be enough, I think:
`return (!testRailCase.Title.Equals(createCaseRequest.Title) || !testRailCase.TemplateId.Equals(createCaseRequest.TemplateId));`
Also, the name of this method suggests that we should return some results of this comparison - in our case it should be something like **CheckTestCaseEquality**
And can you please explain why Titles and TemplateIds of compared cases should not be equal to each other?

Reply:
![image](https://user-images.githubusercontent.com/24970976/130233960-e316e3bc-7f5a-4aa5-aa24-3c7b80fb82d9.png)
In this case, the second condition and a return statement are missed.

Naming fixed.

Titles and TemplateIds just checked for equality.